tmt/cli.py

```diff
@@ -1684,6 +1689,13 @@ def try_command(context: Context, image_and_how: str, **kwargs: Any) -> None:
         else:
             options = {"image": image_and_how}
 
+        # For 'connect' rename 'image' to 'guest'
+        try:
+            if options['how'] == 'connect':
+                options['guest'] = options.pop('image')
+        except KeyError:
+            pass
```

Comment:
with `get` we can avoid the try / except handling?

```suggestion
            if options.get('how') == 'connect':
                options['guest'] = options.pop('image')
```